rest/src/main/java/pro/taskana/rest/RestApplication.java

```diff
@@ -31,6 +37,8 @@
 import pro.taskana.sampledata.SampleDataGenerator;
 
 @SpringBootApplication
+@PropertySource("classpath:datasource.properties")
```

Comment:
may be we change to the default application.properties. we already use the prefix datasource for this.